Dockerfile

```diff
@@ -0,0 +1,26 @@
+# TODO(hjiang): Use release build for docker images, update after official release.
+
+# Build base image for compilation and linking.
+FROM mcr.microsoft.com/devcontainers/rust:latest AS builder
+
+WORKDIR /workspace
+
+COPY . .
+
+RUN apt-get update && apt-get install -y gcc-x86-64-linux-gnu
```

Comment:
GKE pod by default uses x86-64 ISA. At the end of the day, should support parameters.